dvc/command/repro.py

```diff
@@ -50,7 +49,7 @@ def add_arguments(repro_parser):
     repro_parser.add_argument(
         "targets",
         nargs="*",
-        help=f"Stages to reproduce. '{PIPELINE_FILE}' by default.",
+        help="Stages to reproduce. 'dvc.yaml' by default.",
```

Comment:
Like I said above, importing `dvcfile` is slow. And, `dvc.yaml` is set in stone, so no need to import.